src/content/protocols/ethena.md

```diff
@@ -0,0 +1,158 @@
+---
+protocol: "Ethena"
+website: "https://ethena.fi/"
+x: "https://x.com/ethena_labs"
+github: "https://github.com/ethena-labs"
+defillama_slug: ["ethena"]
+chain: "Ethereum"
+stage: 0
+risks: ["L", "L", "H", "H", "M"]
+author: ["andrijdavid"]
+submission_date: "2024-12-10"
+publish_date: "1970-01-01"
+acknowledge_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+
+Ethena is a synthetic dollar protocol built on Ethereum that provides a crypto-native solution for money not reliant on traditional banking system infrastructure. It introduces the 'Internet Bond,' a globally accessible dollar-denominated rewards instrument. Ethena's synthetic dollar, USDe, is fully backed by staked crypto assets (Etherum and Bitcoin) and its corresponding short futures positions. Ethena employs delta-hedging strategies to maintain stability and generate yields for users.
+
+# Overview
+
+## Chain
+
+The Ethena protocol is deployed on Ethereum mainnet.
+
+> Chain score: L
+
+## Upgradeability
+
+Ethena's smart contracts are not upgradable but the ability to set and update roles introduces a level of flexibility and risk. 
+
+> Upgradeability: L
+
+## Autonomy
+
+Ethena's contracts have dependencies on external roles and addresses, such as the minter, redeemer, and custodian addresses. 
```

Comment:
Autonomy section should summarise external dependencies, like oracles, or off-chain services. Please list them here.

Reply:
I think the text from below could be used here, but please elaborate what dependency failure would mean for the system, like loosing peg etc.

```
The Ethena protocol has several critical dependencies that introduce significant risks:

    - Centralized Exchanges (CEX): Essential for maintaining delta-neutral positions, crucial for the stablecoin mechanism.
    - Oracles: Relies on CEX (Binance, Bybit, Okx, Deribit, Bitmex and Bitget), Pyth, and Redstone for accurate and timely price feeds.
    - Off-Exchange Settlement Providers: Depends on services like Copper, Ceffu, and Cobo for secure custody and settlement.
    - Cloud Infrastructure: Deployed on AWS regions in Hong Kong (AP-EAST-1) and Ireland (EU-WEST-1), introducing a dependency on cloud services.
```